fastlane/Fastfile

```diff
@@ -112,7 +112,7 @@ platform :ios do
       sdk: "iphonesimulator",
       configuration: "Release",
       derivedDataPath: directory,
-      xcargs: "ARCHS=\"i386 x86_64\" ONLY_ACTIVE_ARCH=NO",
+      xcargs: "ARCHS=\"x86_64\" ONLY_ACTIVE_ARCH=NO",
```

Comment:
Actually I'm wondering whether we could just remove `xcargs` param entirely and always build for active arch.

Reply:
I think we may need to build for both aarch64 (Apple silicon) + x86_64 (Intel) later this year.